galaxy/Dockerfile

```diff
@@ -19,7 +19,7 @@ GALAXY_CONFIG_DIR=/etc/galaxy
 ENV GALAXY_CONFIG_FILE=$GALAXY_CONFIG_DIR/galaxy.ini \
 GALAXY_CONFIG_JOB_CONFIG_FILE=$GALAXY_CONFIG_DIR/job_conf.xml \
 GALAXY_CONFIG_JOB_METRICS_CONFIG_FILE=$GALAXY_CONFIG_DIR/job_metrics_conf.xml \
-GALAXY_VIRTUALENV=$GALAXY_ROOT/.venv/ \
+GALAXY_VIRTUAL_ENV=/galaxy_venv \
```

Comment:
One more thing that pops into my mind. This one is now not share anymore with the /export dir. This means that if we use a extern cluster the Galaxy venv is not share with the extern cluster. This could break things pretty badly, isn't it?